### DISCUSSION TITLE
Worldle

### DISCUSSION THREAD

**Discussion by jeprecated** [axelboc/anki-ultimate-geography#516] [General]
I am sure most have heard of the internet sensation that is [Wordle](https://www.nytimes.com/games/wordle/index.html). Well someone has made a geography specific version :grin: Worldle! https://worldle.teuteuf.fr/

I hereby challenge you all here to the daily Geography guess :grin: 

Hint: the image shown is indeed the country it wants you to guess. You can disable this, or add a random rotation to the image, in the settings

**Comment by jeprecated** [comment]
#Worldle #21 2/6
🟩🟩⬛⬛⬛
🟩🟩🟩🟩🟩

Got it in 2 :sunglasses:

**Comment by axelboc** [comment]
Haha brilliant! Thanks for sharing 😉

**Comment by axelboc** [comment]
I remember a Facebook game I used to play that showed country outlines like this. Can't remember the name, though. The distance clues are very fun in this Worldle.

**Reply by jeprecated** [reply]
We gotta show off our almost useless skill somehow, aye? 😁

**Comment by axelboc** [comment]
Damn it, so close! 😂 

![image](https://user-images.githubusercontent.com/2936402/153707163-04c32646-15f3-4f8e-bc85-17ced1ea6076.png)

**Reply by aplaice** [reply]
Yeah, that area is tricky!

![image](https://user-images.githubusercontent.com/16461345/153712848-a42ffe8a-fecf-4cd3-b7dc-db5695aa8910.png)

**Reply by jeprecated** [reply]
![image](https://user-images.githubusercontent.com/1471350/153722497-e306a4ad-5d11-4548-83e2-cd320d633dac.png)


OH YEAA BABY!

**Comment by aplaice** [comment]
#Worldle #23 1/6
🟩🟩🟩🟩🟩

:)

**Reply by jeprecated** [reply]
#Worldle #23 1/6
🟩🟩🟩🟩🟩

The images are sometimes too powerful :grin:

**Reply by axelboc** [reply]
Haha yeah, this one was easy 😁

**Comment by jeprecated** [comment]
Nooooo :sob: 

#Worldle #24 X/6
🟩🟩🟩🟩🟨
🟩🟩🟩🟩⬛
🟩🟩🟩🟩🟨
🟩🟩🟩🟩🟨
🟩🟩🟩🟩🟨
🟩🟩🟩🟩🟨

**Reply by axelboc** [reply]
Phewww

#Worldle #24 6/6
🟩🟩🟩🟩⬜
🟩🟩🟩🟨⬜
🟩🟩🟩🟩⬜
🟩🟩🟩🟩🟨
🟩🟩🟩🟩🟨
🟩🟩🟩🟩🟩
https://worldle.teuteuf.fr

**Reply by aplaice** [reply]
(With map.)

#Worldle #24 1/6
🟩🟩🟩🟩🟩

**Comment by jeprecated** [comment]
#Worldle #25 4/6
🟩🟩🟨⬛⬛
🟩🟩🟩🟩⬛
🟩🟩🟩🟩🟨
🟩🟩🟩🟩🟩

**Reply by aplaice** [reply]
Tried without map, was very lucky.

#Worldle #25 3/6
🟩🟩🟩⬜⬜
🟩🟩🟩🟩⬜
🟩🟩🟩🟩🟩

**Reply by jeprecated** [reply]
I think I will try the next ones without the map too 👍

**Comment by aplaice** [comment]
#Worldle #26 4/6 
🟩🟩🟩🟨⬜
🟩🟩🟩🟩⬜
🟩🟩🟩🟩🟨
🟩🟩🟩🟩🟩

**Reply by jeprecated** [reply]
#Worldle #26 X/6 🙈
🟩🟩🟩🟩🟨
🟩🟩🟩🟩🟨
🟩🟩🟩🟩🟨
🟩🟩🟩🟩🟨
🟩🟩🟩🟩🟨
🟩🟩🟩🟩🟨

Nooooooo I couldn't remember that one was there :sob: Got so lucky with my starting location guess, with no map.

**Comment by aplaice** [comment]
(Without map):

#Worldle #27 3/6
🟩🟩🟨⬜⬜
🟩🟩🟩🟨⬜
🟩🟩🟩🟩🟩

**Reply by axelboc** [reply]
#Worldle #27 3/6 🙈
🟨⬜⬜⬜⬜
🟩🟩🟩🟨⬜
🟩🟩🟩🟩🟩
https://worldle.teuteuf.fr

Almost got there on the second one, but decided to be conservative :joy:

**Reply by jeprecated** [reply]
I should really learn distances! :sweat_smile: 

![image](https://user-images.githubusercontent.com/1471350/154551110-811fb3b9-003d-4ee2-bad6-8ac1cd65545b.png)

So since Earth's Circumference is 40,000km, then 20,000km is the highest distance away you could get :thinking:

**Comment by jeprecated** [comment]
Got pretty lucky.

#Worldle #28 3/6 🙈
🟩🟩🟨⬛⬛
🟩🟩🟩🟩⬛
🟩🟩🟩🟩🟩

**Reply by aplaice** [reply]
#Worldle #28 4/6
🟩🟩🟩⬜⬜↙️
🟩🟩🟩🟨⬜↗️
🟩🟩🟩🟩🟨↘️
🟩🟩🟩🟩🟩🎉

**Comment by aplaice** [comment]
Phew...

#Worldle #29 6/6 (100%)
🟩🟩🟩⬜⬜↘️
🟩🟩🟩🟩⬜↖️
🟩🟩🟩🟩⬜↘️
🟩🟩🟩🟩🟨↙️
🟩🟩🟩🟩🟨↖️
🟩🟩🟩🟩🟩🎉

**Reply by jeprecated** [reply]
#Worldle #29 X/6 (97%) 🙈
🟩🟩🟩🟩⬛↘️
🟩🟩🟩🟩🟨↖️
🟩🟩🟩🟩🟨↘️
🟩🟩🟩🟩🟨↗️
🟩🟩🟩🟩🟨↙️
🟩🟩🟩🟩🟨➡️

God damn it! :see_no_evil: :sweat_smile:

**Reply by axelboc** [reply]
#Worldle #29 6/6 (100%) 🙈
🟩🟩🟩🟩⬜⬇️
🟩🟩🟩🟩🟨↖️
🟩🟩🟩🟩🟨↘️
🟩🟩🟩🟩🟨↗️
🟩🟩🟩🟩🟨↘️
🟩🟩🟩🟩🟩🎉
https://worldle.teuteuf.fr

**Comment by jeprecated** [comment]
#Worldle #30 X/6 (96%) 🙈
🟩🟩🟩🟩⬛↗️
🟩🟩🟩🟩⬛⬆️
🟩🟩🟩🟩⬛⬅️
🟩🟩🟩🟩🟨↘️
🟩🟩🟩🟩🟨↗️
🟩🟩🟩🟩🟨↗️

Again! :see_no_evil: this game is helping me realise my mental visualisation of the world is lacking :sweat_smile: I can guess any country, and know where they are. But... just a generic "up and to the left from X" really does not mesh with my knowledge base. This is a very good exercise! :grin:

**Reply by aplaice** [reply]
I was pretty lucky:

#Worldle #30 4/6 (100%) 🙈
🟩🟩🟩🟨⬜↗️
🟩🟩🟩🟩🟨↗️
🟩🟩🟩🟩🟨↖️
🟩🟩🟩🟩🟩🎉

> But... just a generic "up and to the left from X" really does not mesh with my knowledge base. This is a very good exercise! grin

Yeah, definitely! It's especially interesting with distances — it says e.g. 5000 km, and (even if I get the approximate distance right) I usually overshoot by 100% or undershoot by 50%...

**Reply by axelboc** [reply]
Yeah same 😬 -- I find the skewed distances of the Mercator projection are really throwing me off. I undershoot in the southern hemisphere and overshoot in the northern hemisphere, especially in Europe/Asia.

**Comment by aplaice** [comment]
#Worldle #31 3/6 (100%) 🙈
🟩🟩🟩🟩⬜↖️
🟩🟩🟩🟩🟨↖️
🟩🟩🟩🟩🟩🎉

**Reply by axelboc** [reply]
Hang on... something's definitely weird with the directional clues:

![image](https://user-images.githubusercontent.com/2936402/155002965-f8c712a7-5be6-4069-9bd9-a638fad72a80.png)

How is Sweden to the north-west of Sudan and to the south-east of Iceland?? Is Wordle not able of displaying ⬆️ or ➡️ ?

**Reply by aplaice** [reply]
Yeah, that's very weird. Worldle seems capable of displaying ⬆️ and ➡️, but only seems to use it when it's very close to due North/East etc.

Experimenting:
![image](https://user-images.githubusercontent.com/16461345/155006502-386b34ab-3094-4e35-bb91-7975323806b5.png)

I'm not sure how Worldle calculates bearings (the code is obfuscated and AFAIU there are different ways of obtaining bearings), but the coordinates used for Sweden and Sudan are as follows:

```json
{code:"SE",latitude:60.128161,longitude:18.643501,name:"Sweden"}
{code:"SD",latitude:15.8905231,longitude:30.7671933,name:"Sudan"}
```

Assuming that we follow a loxodrome/rhumb line (the only case where the bearing is kept constant throughout the journey from A to B — i.e. AFAICT the only case where the bearing has a single value), I _think_ that we have an angle of 11.5° away from due north, which is definitely closer to N than to NW... (Rhumb lines are straight lines in the Mercator projection (apparently one of the reasons why it was so popular...).)

Hence, I definitely agree that Worldle is wrong!

**Reply by aplaice** [reply]
Oh, digging through the code, I've figured out the "guilty snippet":

```
Sn={N:"\u2b06\ufe0f",NNE:"\u2197\ufe0f",NE:"\u2197\ufe0f",ENE:"\u2197\ufe0f",E:"\u27a1\ufe0f",ESE:"\u2198\ufe0f",SE:"\u2198\ufe0f",SSE:"\u2198\ufe0f",S:"\u2b07\ufe0f",SSW:"\u2199\ufe0f",SW:"\u2199\ufe0f",WSW:"\u2199\ufe0f",W:"\u2b05\ufe0f",WNW:"\u2196\ufe0f",NW:"\u2196\ufe0f",NNW:"\u2196\ufe0f"}
```

All of NNE, NE and ENE are converted to `\u2197\ufe0f` (↗️) etc.

Edit: Googling, it turns out that Worldle is [on GitHub](https://github.com/teuteuf/worldle), so I was digging through the transpiled code unnecessarily...

**Reply by jeprecated** [reply]
PR fix incoming :rofl: make sure and link it to this discussion!

**Reply by jeprecated** [reply]
#Worldle #31 3/6 (100%) 🙈
🟩🟩🟩🟨⬛⬆️
🟩🟩🟩🟩🟨↗️
🟩🟩🟩🟩🟩🎉

**Reply by axelboc** [reply]
Hehe https://github.com/teuteuf/worldle/issues/51

**Reply by aplaice** [reply]
It seems that I was too late. :D (The PR is very close to what I was planning to suggest, though, so I'm happy. :))

**Comment by jeprecated** [comment]
#Worldle #32 4/6 (100%) 🙈
🟩🟩🟩🟨⬛⬆️
🟩🟩🟩🟩🟨⬆️
🟩🟩🟩🟩🟨⬆️
🟩🟩🟩🟩🟩🎉

**Reply by aplaice** [reply]
#Worldle #32 2/6 (100%) 🙈
🟩🟩🟩🟩🟨↗️
🟩🟩🟩🟩🟩🎉

**Comment by aplaice** [comment]
Lucky first guess!

#Worldle #33 3/6 (100%) 🙈
🟩🟩🟩🟩🟨➡️
🟩🟩🟩🟩🟨↖️
🟩🟩🟩🟩🟩🎉

**Reply by jeprecated** [reply]
#Worldle #33 4/6 (100%)
🟩🟩🟩🟩🟨➡️
🟩🟩🟩🟩🟨⬆️
🟩🟩🟩🟩🟨⬆️
🟩🟩🟩🟩🟩🎉

**Comment by jeprecated** [comment]
#Worldle #34 X/6 (97%) 🙈
🟩🟩🟨⬛⬛➡️
🟩🟩🟩🟩⬛↖️
🟩🟩🟩🟩⬛↘️
🟩🟩🟩🟩🟨↘️
🟩🟩🟩🟩🟨⬅️
🟩🟩🟩🟩🟨↘️

Nooooo, the damn distances ruined me again :sob:

**Comment by aplaice** [comment]
Again a very lucky first guess. (I rotate the first guess continent and country, for variety/fun and it's frequently unreasonably accurate. :/)

#Worldle #35 2/6 (100%) 🙈
🟩🟩🟩🟩⬜⬆️
🟩🟩🟩🟩🟩🎉

**Reply by axelboc** [reply]
Haha you made me pick the right continent, thanks :grin: 

#Worldle #35 3/6 (100%) 🙈
🟩🟩🟩🟩⬜➡️
🟩🟩🟩🟩🟨⬅️
🟩🟩🟩🟩🟩🎉

**Reply by jeprecated** [reply]
#Worldle #35 4/6 (100%) 🙈
🟩🟩🟨⬛⬛⬅️
🟩🟩🟩🟩⬛↘️
🟩🟩🟩🟩🟨↖️
🟩🟩🟩🟩🟩🎉

Africa was not the right continent :sweat_smile: feels right guessing the middle though :grin:

**Comment by jeprecated** [comment]
#Worldle #36 4/6 (100%) 🙈
🟩🟩🟩⬛⬛↖️
🟩🟩🟩🟩🟨↖️
🟩🟩🟩🟩🟨↖️
🟩🟩🟩🟩🟩🎉

Cloooooser, cloooser!

**Reply by aplaice** [reply]
#Worldle #36 5/6 (100%) 🙈
🟩🟩🟩🟨⬜⬆️
🟩🟩🟩🟩⬜⬆️
🟩🟩🟩🟩🟨↖️
🟩🟩🟩🟩🟨↘️
🟩🟩🟩🟩🟩🎉

**Comment by jeprecated** [comment]
#Worldle #37 3/6 (100%) 🙈
🟩🟩🟩🟩⬛⬆️
🟩🟩🟩🟩🟨⬆️
🟩🟩🟩🟩🟩🎉

Huh, well wha' d' ya know?

**Reply by aplaice** [reply]
#Worldle #37 5/6 (100%) 🙈
🟩🟩🟩🟨⬜⬆️
🟩🟩🟩🟩🟨➡️
🟩🟩🟩🟩🟨↖️
🟩🟩🟩🟩🟨↗️
🟩🟩🟩🟩🟩🎉

**Comment by jeprecated** [comment]
#Worldle #38 X/6 (96%) 🙈
🟩🟩⬛⬛⬛➡️
🟩🟩🟩🟩🟨⬇️
🟩🟩🟩🟩🟨↙️
🟩🟩🟩🟩🟨↘️
🟩🟩🟩🟩⬛↖️
🟩🟩🟩🟩🟨↗️

Noooo I got 2 mixed up :sob: 

See [this image here](https://user-images.githubusercontent.com/1471350/155936150-556b6553-9c9a-44d3-b017-7111a28575a5.png) (not embedded so no spoilers) for two weird distances - that might actually be right :sweat_smile:

**Reply by axelboc** [reply]
Haha I made very similar guesses and didn't get it either

**Reply by aplaice** [reply]
#Worldle #38 6/6 (100%) 🙈
🟩🟩⬜⬜⬜↘️
🟩🟩🟩⬜⬜➡️
🟩🟩🟩🟩🟨↘️
🟩🟩🟩🟩🟨⬇️
🟩🟩🟩🟩🟨↘️
🟩🟩🟩🟩🟩🎉

Yeah, those distances are weird, but plausible.

**Comment by aplaice** [comment]
#Worldle #39 4/6 (100%) 🙈
🟩🟩🟩🟩⬜⬅️
🟩🟩🟩🟩🟨↘️
🟩🟩🟩🟩🟨⬅️
🟩🟩🟩🟩🟩🎉

<details><summary>Indirect spoiler</summary>
For this one, I regret playing without the map.
<details>

**Reply by jeprecated** [reply]
#Worldle #39 5/6 (100%) 🙈
🟩🟩⬛⬛⬛⬅️
🟩🟩🟩🟨⬛↖️
🟩🟩🟩🟩🟨↘️
🟩🟩🟩🟩🟨↘️
🟩🟩🟩🟩🟩🎉

<details><summary>Super secret reply</summary>
Now THAT is a weird map, indeed! :grin: 
<details>

**Comment by jeprecated** [comment]
#Worldle #40 4/6 (100%) 🙈
🟩🟩🟨⬛⬛↙️
🟩🟩🟩🟩⬛⬆️
🟩🟩🟩🟩🟨⬇️
🟩🟩🟩🟩🟩🎉

**Reply by aplaice** [reply]
#Worldle #40 3/6 (100%) 🙈
🟩⬜⬜⬜⬜↙️
🟩🟩🟩🟩⬜↘️
🟩🟩🟩🟩🟩🎉

**Comment by aplaice** [comment]
#Worldle #41 4/6 (100%) 🙈
🟩🟨⬜⬜⬜⬅️
🟩🟩🟩🟩⬜⬅️
🟩🟩🟩🟩🟨↘️
🟩🟩🟩🟩🟩🎉

**Comment by jeprecated** [comment]
#Worldle #42 4/6 (100%) 🙈
🟩🟩🟩⬛⬛➡️
🟩🟩🟩🟩⬛⬇️
🟩🟩🟩🟩🟨↙️
🟩🟩🟩🟩🟩🎉

<details><summary>Well, well, well...</summary>
This will teach you to remove it from the deck! :wink: 
<details>

**Reply by aplaice** [reply]
#Worldle #42 3/6 (100%) 🙈
🟩⬜⬜⬜⬜➡️
🟩🟩🟩🟩🟨➡️
🟩🟩🟩🟩🟩🎉

**Comment by aplaice** [comment]
#Worldle #43 3/6 (100%) 🙈
🟩🟩🟩⬜⬜↙️
🟩🟩🟩🟩🟨↙️
🟩🟩🟩🟩🟩🎉

**Reply by jeprecated** [reply]
Played on my phone, which still had the map enabled 🙈 

#Worldle #43 1/6 (100%)
🟩🟩🟩🟩🟩🎉

**Comment by jeprecated** [comment]
#Worldle #44 5/6 (100%) 🙈
🟩🟩🟩🟩⬛↘️
🟩🟩🟩🟩🟨↙️
🟩🟩🟩🟩🟨↗️
🟩🟩🟩🟩🟨↘️
🟩🟩🟩🟩🟩🎉

**Reply by aplaice** [reply]
#Worldle #44 3/6 (100%) 🙈
🟩🟩⬜⬜⬜➡️
🟩🟩🟩🟩⬜↘️
🟩🟩🟩🟩🟩🎉

**Comment by jeprecated** [comment]
#Worldle #45 X/6 (90%)
🟨⬛⬛⬛⬛➡️
🟩🟩🟩🟨⬛➡️
🟩🟩⬛⬛⬛↙️
🟩🟩🟩🟩🟨↘️
🟩🟩🟩🟩⬛↘️
🟩🟩🟩🟩⬛↖️

Completely failed.

<details><summary>Interestingly an option you can guess is...</summary>
Micronesia :thinking: not Federated States of, just Micronesia 
<details>

**Reply by aplaice** [reply]
#Worldle #45 4/6 (100%) 🙈
🟩⬜⬜⬜⬜➡️
🟩🟩🟩🟨⬜↘️
🟩🟩🟩🟩⬜⬅️
🟩🟩🟩🟩🟩🎉

<details>
>  Micronesia thinking not Federated States of, just Micronesia 
Interesting. It is slightly weird, like calling the USA America.

I had read up about the various Pacific islands about a week ago, otherwise I'd have failed terribly...
</details>

**Reply by axelboc** [reply]
Got it by looking at the map on the last go... 😂

**Comment by aplaice** [comment]
#Worldle #46 4/6 (100%) 🙈
🟩🟩🟩🟩⬜↘️
🟩🟩🟩🟩🟨➡️
🟩🟩🟩🟩🟨⬅️
🟩🟩🟩🟩🟩🎉

**Reply by jeprecated** [reply]
#Worldle #46 6/6 (100%) 🙈
🟩🟩🟩🟩⬛↗️
🟩🟩🟩🟩🟨⬆️
🟩🟩🟩🟩🟨↗️
🟩🟩🟩🟩⬛↖️
🟩🟩🟩🟩🟨⬅️
🟩🟩🟩🟩🟩🎉
 Phew!

**Comment by aplaice** [comment]
#Worldle #47 6/6 (100%) 🙈
🟩🟩🟩🟨⬜⬅️
🟩🟩🟩🟩🟨⬅️
🟩🟩🟩🟩🟨⬅️
🟩🟩🟩🟨⬜↗️
🟩🟩🟩🟩🟨⬅️
🟩🟩🟩🟩🟩🎉

**Reply by jeprecated** [reply]
#Worldle #47 5/6 (100%) 🙈
🟩🟩🟩🟩⬛⬅️
🟩🟩🟩🟩🟨↘️
🟩🟩🟩🟩🟨↘️
🟩🟩🟩🟩🟨➡️
🟩🟩🟩🟩🟩🎉

**Comment by aplaice** [comment]
#Worldle #48 5/6 (100%) 🙈
🟩🟩🟨⬜⬜➡️
🟩🟩🟩🟩⬜↗️
🟩🟩🟩🟩🟨↖️
🟩🟩🟩🟩🟨↘️
🟩🟩🟩🟩🟩🎉

**Comment by jeprecated** [comment]
#Worldle #49 5/6 (100%) 🙈
🟩🟩🟩🟨⬛↖️
🟩🟩🟩🟩🟨↖️
🟩🟩🟩🟩🟨⬇️
🟩🟩🟩🟩🟩↖️
🟩🟩🟩🟩🟩🎉

<details><summary>100%, lol</summary>

![image](https://user-images.githubusercontent.com/1471350/157820821-a965fe68-f140-4560-9727-2720aa321319.png)

<details>

**Reply by axelboc** [reply]
<details>
I got :arrow_left: of Switzerland as a clue, and knew it wasn't France (because my partner was struggling :joy:), so I was like .... hmm, wat?!

I kinda wished they would only use sovereign countries, but anyway, still fun :smile: 
</details>

**Reply by aplaice** [reply]
#Worldle #49 5/6 (100%) 🙈
🟩🟩🟨⬜⬜⬅️
🟩🟩🟨⬜⬜↖️
🟩🟩🟩🟩⬜⬅️
🟩🟩🟩🟩🟨↖️
🟩🟩🟩🟩🟩🎉

<details>

Yeah, "100%" for very close locations is silly, but if 0% is the antipodes and 100% is the centre of the country itself, then 100% here is technically correct (`100*(1-43/40075) ≈ 99.9`). Perhaps in such cases they should display more significant figures (99.x%)?.

I think I had also got a fake 100%, in the Caribbean, once.)

</details>

**Reply by jeprecated** [reply]
Me: "Funny number, haha"
@aplaice: "But what does it mean?" ![](https://media.giphy.com/media/2H67VmB5UEBmU/giphy.gif)

Never change! 😁 😜

**Comment by jeprecated** [comment]
#Worldle #50 X/6 (88%) 🙈
🟩🟩🟩⬛⬛⬇️
🟩🟩🟩🟩⬛⬇️
🟩🟩🟩🟩⬛↙️
🟩🟩🟩🟩⬛⬆️
🟩🟩🟩🟩⬛➡️
🟩🟩🟩🟩⬛⬇️

....fair. If either of you get it I will be greatly impressed! :sweat_smile:

**Reply by axelboc** [reply]
Well, I didn't 😂

**Reply by aplaice** [reply]
Nope...

#Worldle #50 X/6 (88%) 🙈
🟩🟩⬜⬜⬜↙️
🟩🟩🟩🟨⬜↙️
🟩🟩🟩🟩⬜↙️
🟩🟩🟩🟩⬜➡️
🟩🟩🟩🟩⬜⬆️
🟩🟩🟩🟩⬜➡️

**Comment by aplaice** [comment]
#Worldle #51 4/6 (100%) 🙈
🟩🟩🟨⬜⬜↙️
🟩🟩🟩🟩⬜⬆️
🟩🟩🟩🟩⬜⬇️
🟩🟩🟩🟩🟩🎉

**Reply by jeprecated** [reply]
#Worldle #51 4/6 (100%) 🙈
🟨⬛⬛⬛⬛➡️
🟩🟩🟨⬛⬛↙️
🟩🟩🟩🟩⬛⬇️
🟩🟩🟩🟩🟩🎉

**Comment by jeprecated** [comment]
#Worldle #52 X/6 (96%) 🙈
🟩🟨⬛⬛⬛⬅️
🟩🟩🟩🟩⬛⬅️
🟩🟩🟩🟩🟨➡️
🟩🟩🟩🟩🟨↙️
🟩🟩🟩🟩🟨↗️
🟩🟩🟩🟩🟨➡️

Dammit

**Reply by aplaice** [reply]
#Worldle #52 3/6 (100%) 🙈
🟩🟩🟨⬜⬜⬅️
🟩🟩🟩🟩🟨⬆️
🟩🟩🟩🟩🟩🎉

**Comment by jeprecated** [comment]
#Worldle #53 6/6 (100%) 🙈
🟩🟩🟩🟨⬛↗️
🟩🟩🟩🟩⬛➡️
🟩🟩🟩🟩⬛⬆️
🟩🟩🟩🟩⬛⬅️
🟩🟩🟩🟩🟨↗️
🟩🟩🟩🟩🟩🎉
 PHEW

**Reply by aplaice** [reply]
#Worldle #53 4/6 (100%) 🙈
🟩🟩🟩🟨⬜➡️
🟩🟩🟩🟩⬜➡️
🟩🟩🟩🟩🟨➡️
🟩🟩🟩🟩🟩🎉

**Comment by jeprecated** [comment]
Alright guy's it's been a fun month, but I'm gonna dip out :grin: keep up the good fight! :muscle:
